app/assets/javascripts/components/student/student_comments.es6.jsx

```diff
@@ -0,0 +1,51 @@
+class StudentComments extends Component {
+
+  static get propTypes() {
+    return {
+      comments: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      comments: [],
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        flexFlow: 'column',
+        flex: '1',
+      },
+      title: {
+        marginTop: '12px',
+        fontSize: StyleConstants.fonts.sizes.smaller,
+      }
+    };
+  }
+
+  renderStudentComment(comment, index) {
+    return (
+      <StudentComment comment={comment} key={index} />
+    );
+  }
+
+  renderStudentComments() {
+    return this.props.comments.map(this.renderStudentComment.bind(this))
+  }
+
+  render() {
+    var style = Object.assign(
+      {},
+      this.styles.container
```

Comment:
We don't need to use `Object.assign` if we're not merging anything in (unless you're anticipating more styles in the future)...`